commands/config.go

```diff
@@ -78,6 +78,31 @@ func getValue(path []string, obj interface{}) (interface{}, bool) {
 		return val.Interface(), true
 	} else if val.Kind() == reflect.Struct {
 		return getValue(path[1:], val.Interface())
+	} else if val.Kind() == reflect.Map {
```

Comment:
`else` not needed

Reply:
@iomodo  Here I believe it's needed, since it could also be other types like slice which we don't support yet

Reply:
It's just "go way" :
```golang
if x = y1 {
    do stuff
    return ...
}
if x = y2 {
    do stuff
    return ...
}
```

If you have `return`-s there is no need of `else`.